src/test/java/com/revature/chronicle/NoteServiceTests.java

```diff
@@ -45,15 +45,15 @@ public void shouldReturnAListOfAllNotes(){
     public void shouldReturnANoteById(){
         Note note = new Note(1,"www.note.com","a description",new Date(),new User(), new HashSet<Tag>());
         when(repo.findById(1)).thenReturn(Optional.of(note));
-        Note result = service.findById(1);
+        Optional<Note> result = service.findById(1);
         Assert.assertTrue(note.equals(result));
```

Comment:
Will this test pass? Effectively, we are asserting `Note` is equal to `Optional<Note>`.

Reply:
Yeah, I just saw that too. Sorry, August started lecturing while I was working on the tests and I got distracted. I'm fixing that too and looking for any other other issues before I push up my next update.

Reply:
No problem 👍 I'm on the lookout for the next commit.

Reply:
Hey Will! Tests should be fixed and up to code now. We ran into an issue with the last merge where we had conflicting dependencies that were causing the tests the fail. Had to look into that and talk to the other teams to resolve the dependency error without causing problems on their end, but everything is passing now and we should be good to go!

Reply:
Looks good to me @timmycahill I'll go ahead and merge